templates/_helpers.tpl

```diff
@@ -114,4 +114,34 @@ remainder of arguments
   value: {{ join "," $noProxy | quote }}
 - name: no_proxy
   value: {{ join "," $noProxy | quote }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+`image.ref` constructs a full image reference based on the provided registry, repository, tag, and digest values.
+It only includes parts of the reference that are provided, producing an empty string if the repository is not specified.
+*/}}
+{{- define "image.ref" -}}
```

Comment:
In a followup PR, I'll standardize the other image settings to use this helper function.